tests/Team.py

```diff
@@ -110,6 +110,11 @@ def testRepoPermission(self):
         repo = self.org.get_repo("FatherBeaver")
         self.team.set_repo_permission(repo, "admin")
 
+    def testUpdateTeamRepository(self):
+        repo = self.org.get_repo("test-repo")
+        res = self.team.update_team_repository(repo, "admin")
+        self.assertTrue(res)
```

Comment:
Just ``self.assertTrue(self.team.update_team_repository(repo, "admin"))`` it's readable enough as one line.